app/models/covid_health_card.rb

```diff
@@ -2,47 +2,31 @@
 
 # Ties together FHIR models, HealthCard and Rails to create
 # COVID Health Card IG complianct payloads
-class CovidHealthCard
+class CovidHealthCard < HealthCards::HealthCard
```

Comment:
Using inheritance rather than composition doesn't feel right to me here.

Reply:
As we discussed this afternoon, this will be resolved after we allow for custom health card types. This class exists currently as a shim to connect the existing Rails app to the refactored code.